app/src/main/res/values-en/strings-en.xml

```diff
@@ -31,27 +31,29 @@
     </string-array>
     <string name="exercise">Exercises</string>
     <string name="freeplay">Free play</string>
-    <string name="button_ok">I understand</string>
+    <string name="button_ok">Okay</string>
     <string name="scenario_locked">Unavailable</string>
     <string name="scenario_locked_explanation">This scenario is not yet available. Play the previous scenarios to unlock this one!</string>
     <string name="scenario_explanation_title">Explanation</string>
     <string name="scenario_freeplay">This is the free play mode. Here you can design your own ciruit.</string>
-    <string name="scenario1_explanation">Turn on the light by connection the battery with the lamp.</string>
-    <string name="scenario2_explanation">Add a resistor to the circuit, so the light won\'t break. Add a resistor to the circuit, so the light won\'t break. To add the resistor, you have to remove a wire. This can be done by touching a wire in wire mode. Run the circuit when you are done.</string>
+    <string name="scenario1_explanation">Connect the lamp with the battery, to give the lamp power.</string>
+    <string name="scenario2_explanation">Add a resistor to the circuit, so the light won\'t break. To add the resistor, you have to remove a wire. This can be done by touching a wire in wire mode. Run the circuit when you are done.</string>
     <string name="scenario3_explanation">Add a switch to the circuit, so you can flick the Lamp on and off. Afterwards press the switch to turn the lamp on.</string>
     <string name="scenario4_explanation">Now you understand all components, make your own circuit with 2 lightbulbs and a way to turn the lights off and on.</string>
+    <string name="scenario5_explanation">With this exercise you start with nothing! Make circuit with a switch and a lamp, and ensure that the lamp turns on.</string>
```

Comment:
iets andere tekst? 'You now know how to build a circuit, time to try it out yourself!' ofzo